server/routes/boards-routes.js

```diff
@@ -0,0 +1,64 @@
+import express from 'express';
+import Board from '../models/board';
+import Lifecycle from '../models/lifecycle';
+import { OK, INTERNAL_SERVER_ERROR } from '../configs/httpStatusCodes';
+
+const router = express.Router();
+
+// GET USER BOARD LIST
+router.get('/', async (req, res) => {
+  let boards;
+  try {
+    boards = await Board.find();
+    return res.status(OK).send(boards);
+  } catch (e) {
+    return res.status(INTERNAL_SERVER_ERROR).send(`Internal Server Error ${e}`);
+  }
+});
+// CREATE NEW BOARD
+router.post('/', async (req, res) => {
+  const { name, lifeCycles } = req.body;
+  const createdBy = req.user.id;
+  const members = [
+    {
+      member: createdBy,
+      role: 'SUPERADMIN',
+    },
+  ];
+  const board = { name, members, createdBy };
+
+  let savedLifecycles;
+  try {
+    savedLifecycles = await Lifecycle.insertMany(lifeCycles);
+  } catch (e) {
+    return res.status(INTERNAL_SERVER_ERROR).send(`Error saving lifecycles ${e.message}`);
```

Comment:
just create a wrapper with msg as argument, so that you dont have to set the status everytime

Reply:
what if we have to change the status?

Reply:
what if I want to change the status ?

Reply:
you can create a errorWrapper with default status as internal server error and then you can change the status if you want to.

Reply:
can we use middleware to handle the same? using next